Makefile

```
@@ -34,9 +37,12 @@ endif
 all: manager
 
 # Run tests
-test: generate fmt vet manifests
+test: create-release-dir generate fmt vet manifests helm-lint
```

```suggestion
test: generate fmt vet manifests helm-lint
```

I think we don't need `create-release-dir` dependency here.

If it's actually needed, I'd rather suggest move it to where it is depended. Like if `manifests` depends on `create-release-dir `, add `create-release-dir ` as `manifests`'s dependency.